doc/code/qml_fourier.rst

```diff
@@ -0,0 +1,326 @@
+where the coefficients are non-zero, is important
+for the study of the expressivity of quantum circuits, as described in `Schuld,
+Sweke and Meyer (2020) <https://arxiv.org/abs/2008.08605>`__ and `Vidal and
+Theis, 2019 <https://arxiv.org/abs/1901.11434>`__ -- the more coefficients
+available to a quantum model, the larger the class of functions that model can
+represent, potentially leading to greater utility for quantum machine learning
+applications.
+
+Calculating the Fourier coefficients
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Fourier coefficients can be numerically calculated with the
+:func:`~.pennylane.fourier.fourier_coefficients` function:
+
+.. code::
+
+   >>> from pennylane.fourier import fourier_coefficients
+   >>> coeffs = fourier_coefficients(simple_circuit, len(x), 2)
+   >>> print(np.round(coeffs, decimals=4))
+   [0.5 +0.j 0.  -0.j 0.25+0.j 0.25+0.j 0.  -0.j]
+
+The input to the :func:`~.pennylane.fourier.fourier_coefficients` function are
+the function in question, the length of the input vector, and the maximum
+frequency for which to calculate the coefficients (also known as the *degree*).
+(For a quantum function of
+multiple inputs with varying order, it may be necessary to use a wrapper
+function to ensure the Fourier coefficients are calculated with respect to the
+correct input values.)
+
+Internally, the coefficients are computed using numpy's `discrete Fourier
+transform <https://numpy.org/doc/stable/reference/generated/numpy.fft.fftn.html>`__
+function. The order of the coefficients in the output thus follows the standard
+output ordering, i.e., :math:`[c_0, c_1, c_2, c_{-2}, c_{-1}]`, and similarly
+for multiple dimensions.
+
+.. note::
+
+   If a frequency lower than the true maximum frequency is used to calculate the
+   coefficients, it is possible that `aliasing
+   <https://en.wikipedia.org/wiki/Aliasing>`__ will be present in the
+   output. The coefficient calculator also contains a simple anti-aliasing
+   filter that will cut off frequencies higher than a given threshold. This can
+   be configured by setting the ``lowpass_filter`` option to ``True``, and optionally
+   specifying the ``frequency_threshold`` argument (if none is specified, 2 times
+   the specified degree will be used as the threshold).
+
+
+Fourier coefficient visualization
+---------------------------------
+
+A key application of the Fourier module is to analyze the *expressivity* of
+classes of quantum circuit families. The set of frequencies in the Fourier representation
+of a quantum circuit can be used to characterise the function 
+class that a parametrized circuit gives rise to. For example, if an embedding 
+leads to a Fourier representation with few and low-order frequencies, 
+a quantum circuit using this embedding can only express rather simple periodic functions.
+
+The Fourier module contains a number of methods to visualize the coefficients of the Fourier sum 
```

Comment:
Wait, but they are the coefficients of the sum, right? Maybe we could say `series` here @glassnotes? Since the coefficients belong to integer-valued frequencies?

Reply:
Yes `series` works.